src/panels/config/person/dialog-person-detail.ts

```diff
@@ -115,15 +141,33 @@ class DialogPersonDetail extends LitElement {
               @change=${this._pictureChanged}
             ></ha-picture-upload>
 
-            <ha-user-picker
-              label="${this.hass!.localize(
-                "ui.panel.config.person.detail.linked_user"
-              )}"
-              .hass=${this.hass}
-              .value=${this._userId}
-              .users=${this._params.users}
-              @value-changed=${this._userChanged}
-            ></ha-user-picker>
+            <ha-formfield label="Allow person to login">
+              <ha-switch
+                @change=${this._allowLoginChanged}
+                .disabled=${this._user &&
```

Comment:
Should admins be able to change this for other admins?

Reply:
Yes